tests/unit/lms/services/__init___test.py

```diff
@@ -19,7 +18,6 @@ class TestIncludeme:
             ("h_api", HAPI),
             ("launch_verifier", LaunchVerifier),
             ("grading_info", GradingInfoService),
-            ("lti_outcomes_client", LTIOutcomesClient),
```

Comment:
This only tests the services added by name. I made a note to extend this to include every service, include the ones that are registered by class only.